### DISCUSSION TITLE
Search recovered videos

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#2383] [Q&A]
When we search for videos that were recovered, deleted=true, when we examine them, not all of them were actually recovered. When they are not recovered, the preview window displays metadata text, as in the image below. When the video is playable, the video frames appear, that is, IPED is able to identify this.
I am analyzing a case where I have 956 videos, and I am going with the arrow, looking at them one by one and marking when the video is visible to generate a bookmark.
Is there any way to show only the videos that can actually be seen?

![image](https://github.com/user-attachments/assets/064a3f61-0bde-4b96-9bbb-0926b6532933)

tks
paulo

**Comment by lfcnassif** [comment]
Just use the "Files with Thumbnails" filter at the upper left combobox filter.

**Comment by paulobreim** [comment]
Great!

Thank you!

Paulo

Em qui., 12 de dez. de 2024, 22:33, Luis Filipe Nassif <
***@***.***> escreveu:
